src/frontends/pytorch/src/op/to.cpp

```diff
@@ -35,11 +35,7 @@ OutputVector translate_to(NodeContext& context) {
     } else {
         FRONT_END_OP_CONVERSION_CHECK(false, "Unknown aten::to format");
     }
-    FRONT_END_OP_CONVERSION_CHECK(
```

Comment:
Could you please leave a comment that you ignore those 2 inputs and why.

Reply:
Done